### PR TITLE
Added Travis file

### PR DESCRIPTION
**Pull request by ocramz**
Basic support for CI (e.g. only tests against GHC 7.8.4, it would be good to support 7.10.* and 8 as well).

Addresses #6

### CODE REVIEW

**Comment by ocramz** [comment]
All tests green!

**Comment by denibertovic** [comment]
Hey @ocramz this looks awesome! Thank you. Can you squash the commits into one and I'll be happy to merge this.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+sudo: 
+  false
+
+language: 
+  c
```
Comment (denibertovic) [inline .travis.yml]: btw why is langauge C?

Comment (ocramz) [inline .travis.yml]: `language: c` uses a very lightweight image. I've been using it as a default but maybe there are other good practices now

**Comment by ocramz** [comment]
How do I squash to a single commit? Can't you just use the latest one?

**Comment by thierry-b** [comment]
you can now squash the commits during the merge from github UI:
https://github.com/blog/2141-squash-your-commits

else to squash before the merge you can do a "rebase -i <commit number>" where commit number is the last commit from the parent branch. the you "fixup" all commits and do a "push -f" to your branch.

**Comment by ocramz** [comment]
Thank you @thierry-b , do you like this solution @denibertovic ?

**Comment by denibertovic** [comment]
@ocramz I can squash the commits via the CLI. I'd like the new dockerfile changes to not be included in this PR though as to keep things clean and separate.

**Comment by denibertovic** [comment]
@ocramz Also there's major refactoring happening in the dev branch (and lots of stuff I haven't commited yet) so it's probably not worth adding new features to master. If you'd like you can email me and we can discuss what needs to happen to merge the dev changes.

**Comment by ocramz** [comment]
@denibertovic Sure, I forgot this PR is still open: can you squash up to and including 26ee201 , then perhaps commit your new code so I'll take a look and adjust mine? Thanks

**Comment by denibertovic** [comment]
@ocramz I just merged you're Travis changes to Master. So I'm closing this.

**Comment by denibertovic** [comment]
@ocramz As for your dockerfile changes it's probably worth opening an issue first and and discussing implementation details. (for instance I know there is a "dockerfile" module on hackage that we can perhaps use as is?)

**Comment by ocramz** [comment]
@denibertovic Ah, I hadn't thought about checking on Hackage in this case. Yes, it seems like I was rewriting the same stuff as https://hackage.haskell.org/package/dockerfile-0.1.0.1/docs/Data-Docker.html . It's better to use that, I agree.

**Comment by denibertovic** [comment]
@ocramz I've updated the README with some basic info on the sate of the library and mentioned the rewrite on the dev branch. The changes are quite extensive and it will require a lot of work (still) to get into a finished state, I'd be happy to chat with you if you're interested to help out.

**Comment by ocramz** [comment]
Absolutely! Do keep me up to date, thanks!

**Pull request closed without merging**